src/pudl/analysis/plant_parts_eia.py

```diff
@@ -590,11 +570,11 @@ def __init__(self):
         # for all of the plant parts
         self.id_cols_list = make_id_cols_list()
 
-    def execute(self, gens_mega, plants_eia860, utils_eia860):
+    def execute(self, gens_mega, plants_eia860, utils_eia860) -> pd.DataFrame:
```

Comment:
This and the other changes below are just type annotations and reflowing docstrings.